feat(licensing): update description of behavior after license expiration (#6266)
Kong/docs.konghq.com: app/_src/gateway/licenses/index.md

## Deploying the license file

> * **Hybrid mode deployment:** The license file must be deployed to each control
> plane and data plane node. Applying the license through Kong Admin API to the
> control plane. The control plane distributes the license to its data plane nodes.
> This is the only method that applies the license to data planes automatically.
> * **Traditional deployment with no separate control plane:** License file must

Reviewer: ```suggestion
* **Hybrid mode deployment:** The license file must be deployed to each control
plane and data plane node. Apply the license through the Kong Admin API to the
control plane. The control plane distributes the license to its data plane nodes.
This is the only method that applies the license to data planes automatically.
* **Traditional deployment with no separate control plane:** License file must
```

## Deploying the license file

> {% if_version lte:3.4.x %}

Reviewer: @ms2008 Hi! I added conditional rendering because I wasn't sure if licenses would behave the old way on older versions of Kong. If the new license behavior applies to all Kong versions, I can remove the conditional rendering.

Author: Regarding this section of the license file, please remove the conditional rendering, it applies to all Kong versions.

* **Hybrid mode deployment:** The license file only needs to be deployed to
control plane nodes, which distribute the license to the data planes in their
clusters.

> {% endif_version %}
> {% if_version gte:3.5.x %}

Reviewer: ```suggestion
```

Kong Manager displays a banner with a license expiration warning starting at 15 days before expiration.
Expiration warnings also appear in [{{site.base_gateway}} logs](#license-expiration-logs).

> There is a 30 day grace period after license expiry. During the grace period, the full functionality of the license is available, but a message will be logged once a day, which includes information on when the functionality will no longer be available.
> 
> After the grace period, {{site.base_gateway}} behaves as follows:

Reviewer: ```suggestion
After the license expires, there's a 30 day grace period. During the grace period, the full functionality of the license is available, but a message will be logged once a day, which includes information on when the functionality will no longer be available.

After the grace period, {{site.base_gateway}} behaves as follows:
```

* Kong Manager and its configuration are accessible and may be changed, however any [Enterprise-specific features](/gateway/{{page.kong_version}}/kong-enterprise/) become read-only.

> * The Admin API allows OSS features to continue working and configured Kong Enterprise features to continue operating in read-only mode.

Reviewer: ```suggestion
* The Admin API allows OSS features to continue working and configured {{site.ee_product_name}} features to continue operating in read-only mode.
```

* Proxy traffic, including traffic using Enterprise plugins, continues to be processed as if the license had not expired.

> * There may be some Enterprise features that are still writable, but they may also change later, so please don't rely on this behavior.

Reviewer: ```suggestion
* There may be some Enterprise features that are still writable, but they may also change later, so do not rely on this behavior.
```

The behavior of the different deployment modes is as follows:

> - Traditional: Nodes will be able to restart/scale as needed.
> - Hybrid: Existing DPs or new DPs **can accept** config from a CP with an expired license.
> - DB-less and KIC: New nodes **cannot** come up, restarts will break.

Reviewer: ```suggestion
- **Traditional:** Nodes will be able to restart/scale as needed.
- **Hybrid:** Existing data planes or new data planes **can accept** config from a control plane with an expired license.
- **DB-less and KIC:** New nodes **cannot** come up, restarts will break.
```
